### DISCUSSION TITLE
Bots pick up live grenades

### DISCUSSION THREAD

**Discussion by 4Hangar** [FakeFishGames/Barotrauma#14898] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Bots attempt to clean up dropped live grenades

### Reproduction steps

1. Arm Grenade
2. Drop the Grenade in front of the bot (Don't throw)
3. Laugh as they pick up the grenade and blow themselves up

### Bug prevalence

Just once

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by NilanthAnimosus** [comment]
Closing as fixed and merged. Should be fixed now in a future hotfix.
